maud_extras/meta.rs

```diff
@@ -0,0 +1,50 @@
+extern crate maud;
+
+use maud::{Markup, Render};
+
+/// Generate <title> and meta description.
+///
+/// # Example
+///
+/// ```rust
+/// # #![feature(plugin)]
+/// # #![plugin(maud_macros)]
+/// # extern crate maud;
+/// # extern crate maud_extras;
+/// # use maud::Render;
+/// # use maud_extras::*;
+/// # fn main() {
+/// let markup = meta::Tags {
+///   title: "Meta title",
+///   description: "Meta description",
+/// };
+/// let mut got: String = String::new();
+/// markup.render_to(&mut got);
+/// assert_eq!(got, r#"<title>Meta title</title><meta property="description" name="description" content="Meta description">"#);
+/// # }
+/// ```
+pub struct Tags {
```

Comment:
This seems a bit restrictive, why not a Vector Tags? That way one could put whatever meta-attribute one needs.